### DISCUSSION TITLE
[Feature] Folder Search

### DISCUSSION THREAD

**Discussion by JackBird15** [immich-app/immich#14916] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Please can an option for searching a specific additional library be added! It would be great to say "show all photos from additional library 1". 

Reason: I don't want my photography photos to clutter up my dashboard, and batch uploading them all and being able to still look at them is kinda the goal.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7587
